### ISSUE TITLE
Invalid remote Gitlab url

### ISSUE THREAD

**Issue by natros** [bsideup/IDEA-GitLab-Integration#17]
```
Invalid remote Gitlab url: ssh://git@gitlab.foo.com:4589/foo/rfid.git
java.lang.IllegalStateException: Invalid remote Gitlab url: ssh://git@gitlab.foo.com:4589/foo/rfid.git
	at ru.trylogic.idea.gitlab.integration.utils.GitlabUrlUtil.makeRepoUrlFromRemoteUrl(GitlabUrlUtil.java:36)
	at ru.trylogic.idea.gitlab.integration.actions.GitLabOpenInBrowserAction.makeUrlToOpen(GitLabOpenInBrowserAction.java:63)
	at ru.trylogic.idea.gitlab.integration.actions.RemoteSelectedAction.actionPerformed(GitLabOpenInBrowserAction.java:238)
	at ru.trylogic.idea.gitlab.integration.actions.GitLabOpenInBrowserAction.actionPerformed(GitLabOpenInBrowserAction.java:175)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:182)
	at com.intellij.ide.actions.GotoActionAction$7.run(GotoActionAction.java:275)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:337)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:321)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.f(IdeEventQueue.java:866)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:654)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:381)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

IntelliJ IDEA 15.0.2
Build #IU-143.1184, built on December 7, 2015
JRE: 1.8.0_66-b17 x86_64
JVM: Java HotSpot(TM) 64-Bit Server VM by Oracle Corporation

**Comment by bram-atmire** [comment]
Did you manage to fix this or are you still confronted with this problem?

**Comment by bsideup** [comment]
@bram-atmire unfortunately, it's impossible to get http(!) link from ssh(!) remote. Only with additional configuration somewhere in preferences

**Comment by bsideup** [comment]
Closed as duplicate of #14

**Comment by johnpancoast** [comment]
> @bram-atmire unfortunately, it's impossible to get http(!) link from ssh(!) remote. Only with additional configuration somewhere in preferences

@bsideup, I believe the integrated `Github` plugin does this without configuration, doesn't it? Just curious.
